src/components/ConfigurationManagement/ConfigurationManagement.js

```diff
@@ -886,6 +886,28 @@ export const ConfigurationManagement = ({
     });
   };
 
+  const filterFormState = () => {
+    const filteredStackPipes = formState.stackPipes.filter(
+      stackPipe => stackPipe?.retireDate !== stackPipe?.originalRecord?.retireDate
+      || stackPipe?.originalRecord == null
+    );
+    const filteredUnitStackConfigs = formState.unitStackConfigs.filter(
+      unitStackConfig => unitStackConfig?.endDate !== unitStackConfig?.originalRecord?.endDate
+      || unitStackConfig?.originalRecord == null
+    );
+    const filteredUnits = formState.units.filter(unit => 
+      unit.isToggled === true || 
+      filteredUnitStackConfigs.some(config => config.unitId === unit.unitId)
+    );
+    
+    return {
+      units: filteredUnits,
+      stackPipes: filteredStackPipes,
+      unitStackConfigs: filteredUnitStackConfigs
+    };
+  };
```

Comment:
Can just return the stack/pipes and USCs here:
```suggestion
    
    return {
      stackPipes: filteredStackPipes,
      unitStackConfigs: filteredUnitStackConfigs
    };
  };
```

Reply:
Hi @maxdiebold-erg .     const filteredUnits = formState.units.filter(unit => 
      unit.isToggled === true || 
      filteredUnitStackConfigs.some(config => config.unitId === unit.unitId)
    );  I can delete the "unit.isToggled === true || ". I get an error when I don't include the ending single unit ones. I strongly think we should leave "filteredUnitStackConfigs.some(config => config.unitId === unit.unitId)" This creates a new monitory plant for the unitStackConfigs ending.

Reply:
Good point, I forgot about that check! Then yes, I think removing `unit.isToggled === true ||` is sufficient for the units. However, I now think that logic should be added to the `filteredStackPipes`, too, because there may be cases where a USC is ended but a stack/pipe is not retired, in which case we'll still need to include the stack/pipe in the payload.
```javascript
    const filteredStackPipes = formState.stackPipes.filter(
      (stackPipe) =>
        filteredUnitStackConfigs.some(
          (config) => config.stackPipeId === stackPipe.stackPipeId
        ) ||
        stackPipe?.retireDate !== stackPipe?.originalRecord?.retireDate ||
        stackPipe?.originalRecord == null
    );
```